lib/services/language_model.dart

```diff
@@ -2,23 +2,30 @@ import 'package:flutter/material.dart';
 import 'package:helpwave/services/language_preferences.dart';
 
 class LanguageModel extends ChangeNotifier {
-  String _language = "de";
+  String _shortname = "de";
+  String _name = "Deutsch";
 
   final LanguagePreferences _preferences = LanguagePreferences();
-  String get language => _language;
+  String get shortname => _shortname;
+  String get name => _name;
 
   languageModel() {
     getPreferences();
   }
 
-  set language(String locale) {
-    _language = locale;
-    _preferences.setLanguage(locale);
+  set name(String name) {
+    _name = name;
+    _preferences.setLanguage(name);
+    notifyListeners();
+  }
+
+  set shortname(String shortname) {
+    _shortname = shortname;
     notifyListeners();
   }
```

Comment:
only make one function `setLanguage(String name, String shortname)` since shortname and name always have to be updated together and 2x call of a notifier isn't perfect